test/test.googleauth.ts

```diff
@@ -1113,7 +1113,9 @@ describe('GoogleAuth', () => {
 
 describe('.getApplicationDefault', () => {
   it('should return a new credential the first time and a cached credential the second time',
-     async () => {
+     async function() {
```

Comment:
Lets drop this change and the one below.  If we need to increase the timeouts, we should do that in another PR :)